README.rst

```diff
@@ -1,9 +1,27 @@
 |logo| Pootle
 =============
 
-|chat| |build| |health| |coverage| |requirements|
+.. image:: https://img.shields.io/gitter/room/translate/pootle.svg?style=flat-square
```

Comment:
This change is rather sad to see. I made this alteration so that README.rst could be read by humans as well as machines.  So this revert essentially makes it unreadable again.  Is it not possible to handle the |replace| in the release script?  I do find it odd that pypi doesn't do |replace| since it's just docutils.